### DISCUSSION TITLE
Can excecute status of an activity

### DISCUSSION THREAD

**Discussion by nmocruz** [elsa-workflows/elsa-core#510] [Q&A]
I would like to bind buttons visibility/canclick status on my UI screen to the workflow conditions to execute the certain activity.
Example, disable the publish button if the publish workflow requires certain conditions.

**Comment by sfmskywalker** [comment]
Interesting scenario. In some respect, this is the purpose of the `User Task` activity, which exposes a set of actions that a user might perform. You would display buttons on your UI that reflect these buttons. Upon clicking any of these buttons, your application would then trigger the workflow and provide the user action associated with the clicked button as an input.

That is the theory. In practice, the User Task activity is incomplete, and lacking proper documentation to demonstrate the full story.
I will add it to my list of things to do soon as Elsa 2.0 is out,.
